conclave-cloud-api.yml

```diff
@@ -199,6 +199,110 @@ paths:
         default:
           $ref: "#/components/responses/GenericError"
 
+  /projects/{projectUID}/storage:
+    get:
+      description: Get the files stored in a project.
+      operationId: getStorage
+      tags:
+        - Storage
+      parameters:
+        - $ref: "#/components/parameters/projectUID"
+        - $ref: "#/components/parameters/Functions.List.Limit"
```

Comment:
It's a good practice to define new parameters or make the ones from Functions more generic(ie. a base parameter and then inherit it or just use directly) instead of directly referring them here.